src/Umbraco.Web/HtmlHelperRenderExtensions.cs

```diff
@@ -355,8 +352,10 @@ public static MvcForm BeginUmbracoForm(this HtmlHelper html, string action, stri
                                                IDictionary<string, object> htmlAttributes,
                                                FormMethod method)
         {
-            if (string.IsNullOrWhiteSpace(action)) throw new ArgumentNullOrEmptyException(nameof(action));
-            if (string.IsNullOrWhiteSpace(controllerName)) throw new ArgumentNullOrEmptyException(nameof(controllerName));
+            if (action == null) throw new ArgumentNullException(nameof(action));
+            if (string.IsNullOrWhiteSpace(action)) throw new ArgumentException("Value can't be empty or consist only of white-space characters.", nameof(action));
```

Comment:
Should `action` be renamed to `actionName` to keep it the same as in other methods and `controllerName`?

Reply:
Don't think we should change the public param name at this stage, it's a small thing but don't want to introduce some odd breaking change if someone is relying on that name (at least for now and this PR)

Reply:
Yes, it's minor, but indeed still a breaking change... Will leave this alone for now 👍